docs/precommit.md

```diff
@@ -72,25 +72,25 @@ Git-format-staged requires Python v3 or v2.7. Python is usually pre-installed on
 <!--npm-->
 
 ```bash
-npx husky-init
+npx husky init
 npm install --save-dev git-format-staged
-npx husky set .husky/pre-commit "git-format-staged -f 'prettier --ignore-unknown --stdin --stdin-filepath \"{}\"' ."
+echo "git-format-staged -f 'prettier --ignore-unknown --stdin --stdin-filepath \"{}\"' ." > .husky/pre-commit
```

Comment:
Please use `fs.writeFileSync` to write file.

https://github.com/prettier/prettier/pull/16000#discussion_r1477057609